src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -53,17 +53,14 @@ public void ValidPathExists_ReturnsTrue()
             });
         }
 
-        [Fact]
-        public void PathWithInvalidCharactersAsPath_ReturnsFalse()
+        [Theory MemberData("PathsWithInvalidCharacters")]
```

Comment:
Same question... this compiles?  This looks like a compiler bug that was recently fixed:
https://github.com/dotnet/roslyn/issues/14696
Please add a comma (here and elsewhere) so we're not dependent on the bug.
cc: @jaredpar